examples/with-i18n-next-intl/pages/test.js

```diff
@@ -0,0 +1,33 @@
+import { useTranslations } from 'next-intl'
+import { useRouter } from 'next/router'
+import Code from '../components/Code'
+import PageLayout from '../components/PageLayout'
+
+export default function Test() {
+  const { locale } = useRouter()
+  const t = useTranslations('Test')
+
+  return (
+    <PageLayout title={t('title')}>
+      <p>
+        {t('description', {
+          locale,
+          code: (children) => <Code>{children}</Code>,
+        })}
+      </p>
+      <p>
+        {t('lastUpdated', {
+          lastUpdated: new Date('2021-01-22T15:58:45.567Z'),
+        })}
+      </p>
+    </PageLayout>
+  )
+}
+
+export function getStaticProps({ locale }) {
+  return {
+    props: {
+      messages: locale ? require(`../messages/test/${locale}.json`) : undefined,
```

Comment:
I ask because we should only fetch the current translation, ideally. Correct?

Reply:
Yep, we only fetch the minimum of required messages now. The previous example didn't illustrate that particularly well.